ghost/core/core/server/services/segment/ModelEventsAnalytics.js

```diff
@@ -0,0 +1,63 @@
+const _ = require('lodash');
+const logging = require('@tryghost/logging');
+// Listens to model events to layer on analytics - also uses the "fake" theme.uploaded event from the theme API
+const events = require('../../lib/common/events');
+
+const TO_TRACK = [
+    {
+        event: 'post.published',
+        name: 'Post Published'
+    },
+    {
+        event: 'page.published',
+        name: 'Page Published'
+    },
+    {
+        event: 'theme.uploaded',
+        name: 'Theme Uploaded',
+        // {keyOnSuppliedEventData: keyOnTrackedEventData}
+        // - used to extract specific properties from event data and give them meaningful names
+        data: {name: 'name'}
+    },
+    {
+        event: 'integration.added',
+        name: 'Custom Integration Added'
+    },
+    {
+        event: 'settings.edited',
+        name: 'Stripe enabled',
+        data: {key: 'key', value: 'value'}
+    }
+];
+
+module.exports = class ModelEventsAnalytics {
+    #analytics;
+    #trackDefaults;
+    #prefix;
+    #sentry;
+    #toTrack;
```

Comment:
You can also do `#toTrack = [ ... ]` rather than defining and assigning, not a big deal tho